templates/v2/reset_password/beneficiary/reset_password_form_layout.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'v2/reset_password/beneficiary/reset_password_layout.html.twig' %}
+{% form_theme form 'v2/form_theme/create_user_form_theme.html.twig' %}
+
+{% block homeButton %}
+    <a href="{{ path('reset_password_beneficiary', {id: beneficiary.id}) }}"
```

Comment:
We should do the same back button as RP. It uses the browser history API, and it is the same on every page.